docs/serving/samples/hello-world/helloworld-csharp/helloworld-csharp.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <RootNamespace>helloworld_csharp</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="wwwroot\" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />

```

Comment:
A few things to note here:

Upgrding the `<TargetFramework>` to 2.2 is a good catch. I didn't notice this was out of date since I was just updating the Dockerfile, not the app source code. It's a good idea to do this, but it isn't within the scope of updating the Dockerfile. I didn't notice any issues when running the image locally and sending requests to it.

`<AspNetCoreHostingModel>` is a non-default configuration that couples the app to a particular platform. It should be avoided for this docs example.

The NuGet package `Microsoft.AspNetCore.Razor.Design` doesn't have anything to do with the source code of this example app (it doesn't use Razor at all).